File: output/qwen3-30b-a3b-fp16/code_snippet_33.cs
public static int Factorial(int n)
{
    if (n == 0)
        return 1;
    int result = 1;
    for (int i = 1; i <= n; i++)
        result *= i;
    return result;
}

// Example usage
Console.WriteLine(Factorial(5));  // Output: 120

File: output/qwen2.5-coder-32b-instruct-fp16/code_snippet_33.cpp
#include <iostream>

int factorial(int n) {
    if (n == 0) return 1;
    int result = 1;
    for (int i = 1; i <= n; ++i)
        result *= i;
    return result;
}

int main() {
    std::cout << factorial(5) << std::endl;  // Output: 120
    return 0;
}